x-pack/plugins/canvas/public/components/dom_preview/dom_preview.js

```diff
@@ -12,39 +12,47 @@ export class DomPreview extends React.Component {
   static container = null;
   static content = null;
   static observer = null;
+  static original = null;
```

Comment:
Can we switch all of these from `static` to `this.` since the static ones aren't being used?

Reply:
Yes, as said [here](https://github.com/elastic/kibana/pull/34475#issuecomment-479846070) using `static` puzzled me, kept my change minimal but putting these into a `constructor` on your nudge :-)